ssas/service/server.go

```diff
@@ -257,6 +256,31 @@ func (s *Server) VerifyToken(tokenString string) (*jwt.Token, error) {
 	return jwt.ParseWithClaims(tokenString, &CommonClaims{}, keyFunc)
 }
 
+func (s *Server) VerifyClientSignedToken(tokenString string, trackingId string) (*jwt.Token, error) {
+	keyFunc := func(token *jwt.Token) (interface{}, error) {
+		if _, ok := token.Method.(*jwt.SigningMethodRSA); !ok {
+			return nil, fmt.Errorf("unexpected signing method: %v", token.Header["alg"])
+		}
+		claims := token.Claims.(*CommonClaims)
+		if claims.Issuer == ""{
+			return nil, fmt.Errorf("missing issuer (iss) in jwt claims")
+		}
+		system,err := ssas.GetSystemByID(claims.Issuer)
+		if err !=nil {
+			return nil, err
```

Comment:
In the current code `err` is passed to the user unaltered.  Can we make an informative, but generic message either here or in `tokenV2()`?